src/test/scala/chisel3/iotesters/ChiselPokeSpec.scala

```diff
@@ -0,0 +1,40 @@
+// See LICENSE for license details.
+
+import chisel3._
+import chisel3.iotesters.experimental.{ChiselPokeSpec, ChiselImplicitPokeSpec}
+
+class MyDut extends Module {
+  val io = IO(new Bundle {
+    val in = Input(UInt(8.W))
+    val out = Output(UInt(8.W))
+  })
+  io.out := io.in + 1.U
+}
+
+class MyDutSpec extends ChiselPokeSpec {
+  "MyDut" should "properly add" in {
+    run(new MyDut) {(t, c) =>
```

Comment:
I don't particularly like this syntax, though the previous version where you had to manually synchronize the module type between your DUT, your spec, and your tester classes was completely insane. This at least keeps things very DRY and very concise.